src/DynamoCoreWpf/App.config

```diff
@@ -1,5 +1,9 @@
 ﻿<?xml version="1.0"?>
 <configuration>
+  <appSettings>
+    <add key="MLNodeAutocomplete" value="https://dev.autocomplete.dynamobim.org/autocomplete"/>
+	<add key="token" value="eyJhbGciOiJSUzI1NiIsImtpZCI6IlU3c0dGRldUTzlBekNhSzBqZURRM2dQZXBURVdWN2VhIn0.eyJzY29wZSI6WyJkYXRhOnJlYWQiXSwiY2xpZW50X2lkIjoiN3RpUXl2cWtHdmN5M3BYY0NBTW1LbTlldjFHR0FuZGoiLCJhdWQiOiJodHRwczovL2F1dG9kZXNrLmNvbS9hdWQvYWp3dGV4cDYwIiwianRpIjoiQTBrSjZGem8zNTVQTlFtY0hPNFdUYXhleG9ZMDZVM3YzeWZsU25UMkI0OFU3enpSZ3RJczZ1aTQ5UEZZWXRaRiIsImV4cCI6MTY2ODAwNDA3Nn0.Q_yhMnWR9IyQaKgbDyfEXQgRjDlc57MMVLAhWi-5_n905XxGdzqR5KwZbSVirfrJQ70R-9XaaXbOLIdTZmMGcQlU4D8U3anMsg8O7tXns1HmZ3lRTDQSXW_KpSBw9AWA2CUFOogKmoCa_xz0PMw-hQWoX6Uw_ER_wB9UrAR5P_A_fwhkLhAgCtbR2JrQmiuJ-Pl3YxXTCKtW7y7t_trImgo98k2BnsyI20jiVn6bEuVZJcZEWJGqtgawDECzI67lg4ivUE2m22B2lxpljmLq5sCphOu4LueNbmuZunbieDrFxTvWOQqPNi5-733Y-PIfOWoT5GJ0TG_Uy9mMVKBwcA"/>
```

Comment:
I don't think we should be committing this, this token can be used, maybe use the new Idsdk API to fetch the token.
https://github.com/DynamoDS/Dynamo/blob/8baa17e786dcf506e18c495ec33f164a2c834c53/src/DynamoCore/Core/IDSDKManager.cs#L182

Reply:
Yes, I wil be removing this token and use the oAuth sign -in. Just saw your PR merged.